Sources/Clappr/Classes/Plugin/Core/DVRPlugin.swift

```diff
@@ -0,0 +1,64 @@
+import AVFoundation
+
+open class DVRPlugin: UICorePlugin {
+    
+    open override var pluginName: String {
+        return "dvr"
+    }
+    
+    var minDvrSize: Double {
```

Comment:
Isn't better instead of _minDrvSize_ be a _computed property_ it be a _constant_?